types/plotly.js-dist-min/plotly.js-dist-min-tests.ts

```diff
@@ -0,0 +1,21 @@
+import Plotly from 'plotly.js-dist-min';
```

Comment:
This should probably be `import * as Plotly`.

Reply:
Fixed 😺